tests/test_crypto/test_ethereum.py

```diff
@@ -204,6 +204,7 @@ def test_construct_sign_and_submit_transfer_transaction(
     assert tx != transaction_receipt, "Should not be same!"
 
 
+@pytest.skip
```

Comment:
@DavidMinarsch faucet not working on Ganache

Reply:
Ok, let's open an issue